### PR TITLE
[8062400] Checkmember replacement org and project baseline controls

### PR DESCRIPTION
**Pull request by arvindsingh-cloud**
## Description
</br>
 Brief description of the requested change.

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

**Review by juhi037 — APPROVED** [review]

File: src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1 [hunk 1]
```diff
@@ -515,16 +515,16 @@ class Project: ADOSVTBase
                     $controlResult.AddMessage([VerificationResult]::Passed,"Number of administrators configured are within than the approved limit: $($this.ControlSettings.Project.MaxPAMembersPermissible).");
                 }
 
-                if($TotalPAMembers -gt 0){
-                    $controlResult.AddMessage("Count of Project Administrators: $($TotalPAMembers)")
+                #if($TotalPAMembers -gt 0){
```
Comment (SaikumarRagala) [inline src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1:518]: Why this condition is removed?

File: src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1 [hunk 2]
```diff
@@ -430,12 +430,12 @@ class Project: ADOSVTBase
                     $controlResult.AddMessage([VerificationResult]::Passed,"Number of administrators configured meet the minimum required administrators count: $($this.ControlSettings.Project.MinPAMembersPermissible)");
                  
                 }
-                if($TotalPAMembers -gt 0){
+                #if($TotalPAMembers -gt 0){
```
Comment (SaikumarRagala) [inline src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1:433]: Why this condition is removed?

File: src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Organization.ps1 [hunk 3]
```diff
@@ -1881,7 +1881,7 @@ class Organization: ADOSVTBase
             $responseObj = @([WebRequestHelper]::InvokeGetWebRequest($apiURL));
 
             $AllUsersAccounts =  @()
-            if(($null -ne $responseObj) -and $responseObj.Count -gt 0 -and ([Helpers]::CheckMember($responseObj[0], 'members')))
+            if($responseObj.Count -gt 0 -and ([Helpers]::CheckMember($responseObj[0], 'members')))
```
Comment (SaikumarRagala) [inline src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Organization.ps1:1884]: What if responeObj is null? $responseObj.Count will be error?

File: src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Organization.ps1 [hunk 4]
```diff
@@ -1846,7 +1846,7 @@ class Organization: ADOSVTBase
             $responseObj = @([WebRequestHelper]::InvokeGetWebRequest($apiURL));
 
             $guestAccounts =  @()
-            if(($null -ne $responseObj) -and $responseObj.Count -gt 0 -and ([Helpers]::CheckMember($responseObj[0], 'members')))
+            if($responseObj.Count -gt 0 -and ([Helpers]::CheckMember($responseObj[0], 'members')))
```
Comment (SaikumarRagala) [inline src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Organization.ps1:1849]: What if responeObj is null? $responseObj.Count will be error?

File: src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1 (hunk 1 above)
Comment (arvindsingh-cloud) [reply src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1:518]: same already checked above and this line comes under that

File: src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1 (hunk 2 above)
Comment (arvindsingh-cloud) [reply src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1:433]: same already checked above and this line comes under that

File: src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Organization.ps1 (hunk 4 above)
Comment (arvindsingh-cloud) [reply src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Organization.ps1:1849]: $responseObj will never come null due to @(); at line 1846

**Review by SaikumarRagala — APPROVED** [review]

**Pull request merged**